ui/src/db-tables/WorkspaceDB.ts

```diff
@@ -97,15 +99,55 @@ export async function backfillIndexdb() {
   const backfillTags = async () => {
     try {
       const all = await tagsTable?.getRecords();
-      all && (await indexdb.tags.bulkAdd(Object.values(all)));
+      const tagList = all ? Object.values(all) : [];
+      // ID is a new field, compatible with the historical data of existing users.
+      if (tagList.length > 0 && !tagList[0].id) {
+        tagList.forEach((tag) => {
+          tag.id = uuidv4();
```

Comment:
oh id is just the same as name you can just add tag.id = tag.name

Reply:
done